apps/meteor/ee/server/models/raw/WorkspaceCredentials.ts

```diff
@@ -0,0 +1,68 @@
+import type { RocketChatRecordDeleted, IWorkspaceCredentials } from '@rocket.chat/core-typings';
+import type { IWorkspaceCredentialsModel } from '@rocket.chat/model-typings';
+import type { Collection, Db, DeleteResult, Filter, IndexDescription, UpdateResult } from 'mongodb';
+
+import { BaseRaw } from '../../../../server/models/raw/BaseRaw';
+
+export class WorkspaceCredentialsRaw extends BaseRaw<IWorkspaceCredentials> implements IWorkspaceCredentialsModel {
+	constructor(db: Db, trash?: Collection<RocketChatRecordDeleted<IWorkspaceCredentials>>) {
+		super(db, 'workspace_credentials', trash);
+	}
+
+	protected modelIndexes(): IndexDescription[] {
+		return [{ key: { _id: 1, scopes: 1, expirationDate: 1, accessToken: 1 }, unique: true }];
+	}
+
+	getCredentialByScope(scope: string): Promise<IWorkspaceCredentials | null> {
```

Comment:
```suggestion
	getCredentialByScope(scope = ''): Promise<IWorkspaceCredentials | null> {
```

This way, you don't have to pass `''` on all the calls to get/unset credentials